pyfolio/perf_attrib.py

```diff
@@ -21,13 +21,19 @@
 import matplotlib.pyplot as plt
 
 from pyfolio.pos import get_percent_alloc
-
 from pyfolio.txn import get_turnover
 from pyfolio.utils import print_table, configure_legend
+from .deprecate import deprecated
 
 PERF_ATTRIB_TURNOVER_THRESHOLD = 0.25
 
+DEPRECATION_WARNING = (
+    "Performance attribution in pyfolio is deprecated. "
+    "Please use empyrical's performance attribution instead."
+)
+
 
+@deprecated(msg=DEPRECATION_WARNING)
```

Comment:
Instead of keeping two separate copies of this logic, should we call into empyrical here instead?  I don't expect that all of our code that uses this will get migrated immediately, so if we make changes/bugfixes somewhere, it'd be nice to not have to update in two places.